projects/wpdx/src/app/rehab-prio/rehab-prio.component.ts

```diff
@@ -788,6 +766,10 @@ export class RehabPrioComponent implements OnInit {
           ]);
         }
       } else {
+        for (const layer of ['nc-points', 'nc-labels', 'nc-heatmap-clustered', 'nc-heatmap']) {
```

Comment:
This part of the if will never happen, as `minConstructionFilt` will never be 'false'-y - in javascript `[]` resolves to `true`.